frontend/src/components/grantaccess/UserIdentityCard.vue

```diff
@@ -28,21 +30,21 @@ const props = defineProps<{
         <template #content>
             <div class="col" style="margin-left: 2rem">
                 <label class="row">Username</label>
-                <span class="row">{{ props.userIdentity.userId }}</span>
+                <span class="row" id="userId">{{ props.userIdentity.userId }}</span>
             </div>
             <div class="col" v-if="props.userIdentity.found">
                 <label class="row">First Name</label>
-                <span class="row">{{ props.userIdentity.firstName }}</span>
+                <span class="row" id="firstName">{{ props.userIdentity.firstName }}</span>
             </div>
-            <div class="col" v-if="props.userIdentity.found">
-                <label class="row">Last Name</label>
+            <div class="col-2" v-if="props.userIdentity.found">
+                <label class="row" id="lastName">Last Name</label>
                 <span class="row">{{ props.userIdentity.lastName }}</span>
             </div>
             <div
                 class="col-6 d-flex"
                 v-if="!props.userIdentity.found"
             >
-                <span class="px-0 invalid"> User does not exist </span>
+                <span class="px-0 invalid" id="userNotExist"> User does not exist </span>
```

Comment:
I'm fine to add ids this time, but normally we don't add id just because of testing. We could just verify the expect the text is rendered. And in the case we want it to be more specific, could also find by span and row and go through the return list. But normally as long as can find the text should be fine.